DataFormats/Common/interface/DetSetVectorNew.h

```diff
@@ -23,6 +23,8 @@ namespace edm { namespace refhelper { template<typename T> struct FindForNewDetS
 namespace edmNew {
   typedef uint32_t det_id_type;
 
+  struct CapacityExaustedException {};
```

Comment:
Please use an standard CMS exception so the error reporting system can properly report the problem if it happens.